gitlab/v4/objects/groups.py

```diff
@@ -194,6 +194,21 @@ def delete_ldap_group_link(
         path += f"/{cn}"
         self.manager.gitlab.http_delete(path, **kwargs)
 
+    @cli.register_custom_action("Group")
+    @exc.on_http_error(exc.GitlabGetError)
+    def list_ldap_group_links(self, **kwargs):
+        """List LDAP group links.
+
+        Args:
+            **kwargs: Extra options to send to the server (e.g. sudo)
+
+        Raises:
+            GitlabAuthenticationError: If authentication is not correct
+            GitlabGetError: If the server cannot perform the request
+        """
+        path = "/groups/%s/ldap_group_links" % self.get_id()
```

Comment:
@JohnVillalovos @nejch A general question. Would it make sense to change the syntax such that f-string is being used instead of a string?

Reply:
Yes it would, since @nejch did a big PR to switch everything to f-strings 😊

Reply:
As a note, the `pylint` check should flag non f-string usage.

https://pylint.pycqa.org/en/latest/user_guide/checkers/features.html   consider-using-f-string (C0209)

Reply:
Agreed, CI will take care of this :)

Reply:
rebased off main, replaced with f string, and did the function annotation thing.